ES/ESmy005.xml

```diff
@@ -30,103 +30,2045 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
-                     <title ref="LIT2384Taamme">Taʾāmmǝra Māryām "Miracles of Mary"</title>
+                     <locus from="5ra" to="108vc"/>
+                     <title ref="LIT2384Taamme">Taʾāmmǝra Māryām "Miracles of Mary" (a collection of
+                       153 stories)</title>
                      <textLang mainLang="gez"/>
+
                      <msItem xml:id="ms_i1.1">
                         <locus from="5ra" to="6va"/>
                         <title ref="LIT1968Mashaf"/>
+                        <textLang mainLang="gez"/>
+                        <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡
+                          ቅዱስ፡ ፩</hi>አምላክ፡ ዝንቱ፡ መጽሐፈ፡ ሥርዓት፡ ዘወፅአ፡ እመንበረ፡ ማርቆስ፡
+                          ሐዋ<hi rend="rubric">ርያ፡ እመካነ፡ መዓልቃ፡ ዘምስር፡</hi>
+                       </incipit>
                      </msItem>
+
                      <msItem xml:id="ms_i1.2">
                         <locus from="6va" to="7ra"/>
                         <title type="complete" ref="LIT3058RepCh338"/>
                         <textLang mainLang="gez"/>
+                        <incipit xml:lang="gez">
+                        <hi rend="rubric">እሰግድ፡ ለኪ፡ እሰግድ፡ ለኪ፡ እሰግድ፡ ለኪ፡ ኦእግዝ</hi>እትየ፡ ቅድስት፡
+                        ድንግል፡  በ፪<hi rend="rubric">ማርያም፡</hi> ወላዲተ፡ አምላክ፡
+                        በከመ፡ ወደሰኪ፡ ዮሐንስ፡ አፈ፡ ወርቅ፡ ብድርሳኑ፡
+                        </incipit>
                      </msItem>
+
                      <msItem xml:id="ms_i1.3">
-                        <locus from="7ra" to="8rc"/>
+                        <locus from="7ra" to="7vc"/>
                         <title ref="LIT3584Intro"/>
+                        <textLang mainLang="gez"/>
+                        <incipit xml:lang="gez">
+                        <hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡</hi><gap reason="ellipsis"/> ስምዑ፡ እነግረክሙ፡
+                        አኃውየ፡ ፍቁራን፡ ውሉደ፡ ቤተ፡ ክርስቲያን፡ እግዚ<hi rend="rubric">አብሔር፡ ይባርክሙ፡ ወያብርህ፡ አዕይንተ፡ አል</hi>ባቢክሙ፡
+                        </incipit>
+                         </msItem>
+
+                         <msItem xml:id="ms_i1.4">
+                            <locus from="7vc" to="8rc"/>
+                            <title>Salāmāt-greetings to John the Evangelist,
+                               be read after the Miracles</title>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez">
+                            <hi rend="rubric">እምድኅረ፡ ተፍጻሜተ፡ ተአምሪሃ፡</hi>፡ ይቤሉ፡ ዘንተ፨
+                            <hi rend="rubric">ሰላም፡ ለዮሐንስ፡ ወልደ፡</hi> ነጐድጓድ፨ ሰላም፡ ለከ፡ ዮሐንስ፡ ወልደ፡
+                            አቡቀለምሲስ፨ ሰላም፡ ለከ፡ ዮሐንስ፡ ቴዎሎጎስ፤
+                            </incipit>
+
+                         </msItem>
+
+
+                     <msItem xml:id="ms_i1.5">
+                      <locus from="8rc" to="108vc"/>
+                      <title type="complete" ref="LIT3585Taamme">Miracles of Mary: 153 miracles</title>
+
                      </msItem>
```

Comment:
nesting error:
this `</msItem>` should be moved to the end of `ms_i1.5`, after all the miracles, before the very last `</msItem>`

Reply:
Sorry I do not understand, if you mearn Salāmāt-greetings to John the Evangelist, why should it be after all miracles?

Reply:
I mean the nesting is wrong.
Should be
Item 1.5 opens
     Item 1.5.1
     Item 1.5.3
    Item 1.5.4
      Etc etc
    Item 1.5.153 closed
Item 1.5 closed

Now Item 1.5 is closed before 1.5.1

Reply:
Like what I did?